test/src/test/java/org/corfudb/infrastructure/ServerContextBuilder.java

```diff
@@ -33,20 +31,20 @@
     String truststore = "";
     String truststorePasswordFile = "";
 
-    String implementation = "local";
+    ChannelImplementation implementation = ChannelImplementation.LOCAL;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/annymartinez.m/CorfuDB/pullRequest?prid=4651442)